pygmt/datasets/tile_map.py

```diff
@@ -16,6 +16,11 @@
     TileProvider = None
     _HAS_CONTEXTILY = False
 
+try:
```

Comment:
Style check fails, and ruff recommends:
```
with contextlib.suppress(ImportError):
    import rioxarray
```
but then ruff formats it to:
```
with contextlib.suppress(ImportError):
    pass
```
which makes no sense. 

```suggestion
try:  # noqa: SIM105
```